python/raf/frontend/pytorch.py

```diff
@@ -80,24 +77,38 @@ def inner(model, input_type, input_shape):
             comm = dist.get_communicator()
             device = "cuda:" + str(comm.local_rank)
 
-        if input_type.startswith("float"):
-            input_data = torch.randn(input_shape, dtype=getattr(torch, input_type), device=device)
-        else:
-            assert input_type.startswith("int64"), "Unsupported input type %s" % input_type
-            input_data = torch.randint(10000, input_shape, device=device)
-
+        example_inputs = ()
+        for input_name, input_info in list(shape_dict.items()):        
```

Comment:
No need to make a list when iterating a generator.
```suggestion
        for input_name, input_info in shape_dict.items():
```